x-pack/plugins/enterprise_search/server/routes/enterprise_search/connectors.ts

```diff
@@ -30,6 +33,35 @@ export function registerConnectorRoutes({ router }: RouteDependencies) {
         const body = await addConnector(client, request.body);
         return response.ok({ body });
       } catch (error) {
+        if (error.statusCode === 403) {
+          return createError({
+            errorCode: ErrorCode.UNAUTHORIZED,
+            message: i18n.translate(
```

Comment:
Is this message ever displayed? it seems like in your client logic you check the error code and then display text you define there.

Reply:
It is! It's flashed on the screen.